### ISSUE TITLE
Syriac script in Ethiopic manuscript and its implications

### ISSUE THREAD

**Issue by CarstenHoffmannMarburg** [BetaMasaheft/Documentation#2377]
@eu-genia @thea-m @DenisNosnitsin1970 @AaronButts 

I found one line of Syriac letters in the Florentine manuscript BMLor148 on f. 182r, that I encoded as an additio. It seems to be a great curiosity and I think, it means, that this manuscript has been most likely produced not in Ethiopia, but in Italy. Other witnesses with both Syriac and Gəʿəz script are polyglots that have been produced and reproduced in exclusively in Europe. 

A second indication for the European origin of this manuscript is the fine repair on folio 169 without textloss, what means, that someone repaired this folio with a very thin needle before the text was written on the parchment. In the estimated 15th century, this was unlikely to happen in Ethiopia, because as far as I know, thin needles have been rather unaccessible in these times in inner Africa. 

Do you think these arguments are sufficient to claim an European origin in the record?

The Syriac letters are unconceivable to me. As far as I can decipher them, they are: 
ܬܢ ܬܝܣܚܩ ܥܒܪܠܒܚܜܝܐ ܦܫܐܡܟ
I can recognise with some phantasy the name Yisḥaq ܝܣܚܩ relatively to the beginning, but this is all. The rest seems to me language mash, that cannot be translated. I categorized it as "Unclear" in desc type. Does anybody else have an idea, what phrase may be hidden in this clause? (NB: I would like to present the folio in this issue, but I have not been able to cut and paste folios from the viewer into github. If anyone can explain to me, I would appreciate very much.)

The third issue is the way of encoding Syriac. In this issue, I was able to write Syriac letters from right to left, as it is supposed to be. But in Oxygen, the Syriac letters are presented from left to right and without letters connected to each other. I encoded them isolated from each other and "from back to front" so to say. Does anybody know, how this will be displayed on the website in the end? When I press the butten "Im Browser/In Systemanwendung anwenden" (fifth button in the taskbar, next to the "Save"-button), then nothing happens.

Is it possible to forward this issue also to Ted Erho? I have not seen his name in the list of possible Beta Masaheft reviewers or assignees.

**Comment by CarstenHoffmannMarburg** [comment]
It is perhaps worth mentioning that the Syriac clause has, in my estimation, no relation to the Ethiopian text.

**Comment by DenisNosnitsin1970** [comment]
At least to a part of your question: this manuscript is a very well known one, extensively used by A. Bausi in Sinodos edition, and in other publications too (it contains important texts also beyond Sinodos). Didn't Marrassini (or anyone else) already transcribe the Syriac and took a statement to it? Would be nice to know what it means. The manuscript is on parchment, therefore, the chance that it was copied outside of Ethiopia is not really high. But  since it travelled a lot, the line could be copied into it on various ways.

**Comment by CarstenHoffmannMarburg** [comment]
I have not found any note concerning the Syriac note in Marrassinis record so far (but I have not carefully worked through every detail so far), but there is some information on its origin in record 16.IV (page 92), stating, that it was produced in 1426 according to a notice in a colophon. I will check and include this information into the BM record later on. If the Syriac phrase makes no sense, Marrassini probably regarded it as not important.

**Comment by AaronButts** [comment]
Thanks, Carsten, for including me in this thread! I will also forward to Ted. 

A couple of points in no particular order:

BeInf is aware of this example, and I have actually been working on it! 

Personally I wouldn't necessarily connect this example with the polyglots. Some of the polyglots of course also have  both Syriac and Gəʿəz script, but they are a very different phenomenon: more than these two languages and full texts in parallel, whereas the Syriac in this manuscript is something very different. By the way, there are other examples of this phenomenon of a little bit of Syriac appearing in a fully Ethiopic manuscript, even if the phenomenon is admittedly very rare. 

All that to say, I am not personally convinced that a comparison with the polyglots adds any evidence for the provenance question, whether of the manuscript as a whole or of the Syriac in the manuscript. I don't have answers (yet) myself, but I only note that I don't find the comparison with the polyglots particularly compelling. 

As for the Syriac text, it is very ill formed in my view, from the letter forms to the grammar. I would propose that it means something like "Isaac, the man in sin, forever", which would be:

ܐܝܣܩܚ ܓܒܪܐ ܒܚܛܝܐ ܥܠܡܝܢ

But there are all sorts of problems here. Many of the letters are illformed (such as the three humps of the ܚ instead of two; at the beginning twice initial ܐ is written as ܬ; the ܠ in ܥܠܡܝܢ should connect; etc., etc. -- the list is long). Also, I am still not entirely sure what to make of the initial two letters, written ܬܢ , as you say, but probably for ܐܢ (given the spelling of Isaac just after). Any chance this is Arabic inna? Finally, for me each of the individual words is somewhat clear, and they are common in such notes (or whatever this is), but they don't exactly fit together properly here. Perhaps someone, who doesn't know much Syriac, is copying some Syriac from a different source into this manuscript, imitating what he sees? All that to say, much is uncertain. I have been working on this note for several months now, and I will eventually publish something on it, in the context of BeInf. 

How much detail do you need to include in Bm at this stage? Basically, I am not sure how much sense it makes to include the Syriac text and translation without commentary explaining what is going on. For instance, if something like my text and translation are provided, it is entirely misleading, because that is obviously not what is exactly is written. But there is not really a way to transcribe what is written: ill-formed letters, etc. (for instance, would one write xet and yod for the ill-formed xet?).

I am not familiar with Bm policies and practices in this matter, but I would suggest less is better here, at least until a full study is published. But, I am curious to hear what others think!

**Comment by AaronButts** [comment]
One additional point:  At least as I recall, the polyglots were not all produced in Europe.  In fact weren't many of them brought from Egypt? Especially the Wadi Natrun, where it is known that monks from many different linguistic backgrounds lived in the thirteenth and fourteenth centuries. I believe that this is the case with the three famous pentaglot manuscripts, ms. Vat. Barberini Or. 2 (Psalter), Ambrosianus B20/A inf. (Paul plus Acts), and Oxford, Bodleian Copt. c. 2 (fragments of gospels). All of them as I recall come from Egypt. I may be misremember but I am quite sure at least that there is a lot of evidence that the pentaglot Psalter comes from Abba Macarius in Egypt. 

So, I am again hesitant about the European connection, based on evidence from the polyglots.

**Comment by CarstenHoffmannMarburg** [comment]
I see: I will encode it as an extra (e) with the option to select the type "doodle" but not as an addition (a).

**Comment by AaronButts** [comment]
Happy to help! 

I am still not sure how to include a reading of this Syriac phrase in a way that makes sense for a reader (especially one who might not have any Syriac). If something like my text and translation are provided, it is misleading, because that is not really what is written. But there is not really a way to transcribe what is written: ill-formed letters, etc. (for instance, would one write xet and yod for the ill-formed xet?). So there really needs to be some commentary, etc., but Bm is probably not the place for that. Once I have published my comments on it, then it would be possible to reference that "for further details". But I am not sure how to handle the matter before then.

**Comment by CarstenHoffmannMarburg** [comment]
@AaronButts
I think, I will propose just a small note with the Syriac text as it is discernable from the manuscript next to your emendation and translation. Once your contribution it in pub, we can add a reference. I will include you to the pull request.

**Comment by DenisNosnitsin1970** [comment]
I would suggest not to make an extensive commentary about smth that is uncertain, or requires a very long commentary and multiple interpretations. Since this note is going to be discussed in a publication, opefully, I personally would not try to transcribe it (a transcription may be easily be considered as statement and claim) but would add a reference to the publication instead. It seems that absolutely certain is here only one fact that the letters are ill-shaped. One of the possible interpretations in similar cases might be that someone who did not know Syriac and its script tried to copy those words just imitating the letters as pictures, without knowning how to write them properly. Could it be the case? (But why someone? This is already a difficult question).

**Comment by AaronButts** [comment]
Similar to Denis (as I understand him), I personally prefer a minimalist approach in this case: for various reasons I think it would be best to avoid a transcription. Is it possible simply to say: "Line in ill-formed Syriac"? Then we can add a note to the publication, if/when it appears. I worry that any transcription will be misleading and problematic. 

Indeed, as Denis proposes, I think there is a decent chance that the Syriac here is written by someone who does not (completely) know Syriac, but is simply trying to copy/imitate some words that he has at hand. In any case, this is not the writing of someone who knows Syriac well. And imitation seems to be likely, at least that is what I am planning to propose in the future publication.

**Comment by AaronButts** [comment]
My article on the line of Syriac in this manuscript has now appeared: 

https://hugoye.bethmardutho.org/article/hv27n1butts

There you will find my reading of the Syriac text as well as many comments about the manuscript (trying to account for the sizeable body of secondary literature on this particular manuscript).

**Comment by karljonaskarlsson** [comment]
My updates have now been merged and appear online, but the Syriac appears erroneously on the website (clearly because my encoding didn't interact well with the right-to-left direction of the Syriac script). @eu-genia , @CarstenHoffmannMarburg and @AaronButts , do you think there is a good way to solve this? Or shall I transcribe the consonant text (might be the easiest work-around)?


Screenshot:

<img width="632" alt="Skärmavbild 2024-11-12 kl  11 34 19" src="https://github.com/user-attachments/assets/54612bfc-3a5a-4df6-afa8-2d4df0f0f9d9">

**Comment by CarstenHoffmannMarburg** [comment]
I do not know, why the code did not work out.

**Comment by eu-genia** [comment]
T

**Comment by eu-genia** [comment]
There are three issues here

- [x] (1) the element surplus is not properly enabled in viewItem.xqm (task for me)

- [ ] (2) the encoding as it is now 

```
<desc type="Unclear">A line of ill-shaped Syriac characters
                              (reproduced below according to the reading proposed in
                              <bibl><ptr target="bm:Butts2024Syriac"/></bibl>): 
                              <foreign xml:lang="syr">
                                 <supplied reason="omitted" resp="AMB">ܐ</supplied>ܐܢ
                                 ܐܝܣܚܩ
                                 ܓܒܪܐ
                                 ܚܛܝܐ<surplus resp="AMB">ܒ</surplus>
                                 ܐܡܝܢ<surplus resp="AMB">ܝܥ</surplus>
                              </foreign></desc>
```
cannot work. For quotations we need `q` outside of `desc`, and notes/comments are `note`, so it should have been
```
<desc type="Unclear">A line of ill-shaped Syriac characters</desc>
 
                              <q xml:lang="syr">
                                 <supplied reason="omitted" resp="AMB">ܐ</supplied>ܐܢ
                                 ܐܝܣܚܩ
                                 ܓܒܪܐ
                                 ܚܛܝܐ<surplus resp="AMB">ܒ</surplus>
                                 ܐܡܝܢ<surplus resp="AMB">ܝܥ</surplus>
                              </q>
<note>
                            reproduced according to the reading proposed in
                              <bibl><ptr target="bm:Butts2024Syriac"/></bibl></note>
```
@karljonaskarlsson and @CarstenHoffmannMarburg please fix in a new PR

- [ ] (3) there seems to be some conflict for RTL languages and the LTR tags, maybe TEI Publisher already has a solution @HelenaSabel ?

**Comment by eu-genia** [comment]
Since this has not been properly resolved it should stay open
